ui/app/mirrors/create/cdc/columnbox.tsx

```diff
@@ -20,69 +21,150 @@ export default function ColumnBox({
   rows,
   setRows,
   disabled,
+  showOrdering,
```

Comment:
```suggestion
  showNullable,
```
(rename on the caller too as it wasn't used)